entropy-service/Dockerfile

```
@@ -1,60 +1,87 @@
+    && apt-get install -y --no-install-recommends --no-install-suggests nodejs \
+    && apt-get purge -y --auto-remove \
+       gnupg \
+    && apt-get clean \
+    && rm -rf /var/cache/apt/archives /var/lib/apt/lists/*
 
-RUN apt-get update && apt-get install -y nodejs --no-install-recommends && apt-get clean && rm -rf /var/lib/apt/lists/*
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
 
 RUN npm install -g pm2
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)